staging/src/k8s.io/sample-apiserver/pkg/cmd/server/start.go

```diff
@@ -139,6 +139,10 @@ func (o *WardleServerOptions) Config() (*apiserver.Config, error) {
 	serverConfig.OpenAPIConfig.Info.Title = "Wardle"
 	serverConfig.OpenAPIConfig.Info.Version = "0.1"
 
+	serverConfig.OpenAPIV3Config = genericapiserver.DefaultOpenAPIV3Config(sampleopenapi.GetOpenAPIDefinitions, openapi.NewDefinitionNamer(apiserver.Scheme))
```

Comment:
I know that there are a few places where we use the openapi config to set-up something in the tests, especially in server-side apply tests. Does this mean that the set-up will more complicated, or are we actually using that function? I can't remember.

Reply:
We could add in a step to populate the V3 config with the v2 one if V3 is nil.

Reply:
The set up isn't exactly more complicated, the same steps we do for V2 will just have to be applied to V3. During the transition phase, I'd imagine we want to have something like SSA tested with both V2 and V3. This is probably something for post beta though since we don't have any consumers of OpenAPI v3 yet.